NAM/dsp.cpp

```diff
@@ -23,15 +23,15 @@ DSP::DSP()
 {
 }
 
-DSP::DSP(const double loudness)
+DSP::DSP(const NAM_SAMPLE loudness)
 : mLoudness(loudness)
 , mNormalizeOutputLoudness(false)
 , _stale_params(true)
 {
 }
 
-void DSP::process(double** inputs, double** outputs, const int num_channels, const int num_frames,
-                  const double input_gain, const double output_gain,
+void DSP::process(NAM_SAMPLE** inputs, NAM_SAMPLE** outputs, const int num_channels, const int num_frames,
+                  const NAM_SAMPLE input_gain, const NAM_SAMPLE output_gain,
```

Comment:
Yeah, it looks weird having all of these parameters being called "samples". Can you point me to somewhere else that follows this convention? It just doesn't sound right.

Reply:
I wasn't sure either, and I'm fine either way. "loudness" is an internal parameter anyway, and "input_gain" and "output_gain" should be probably be removed since the plugin just passes "1.0" and handles gain itself (as does my LV2 plugin).